### DISCUSSION TITLE
7가지 폐질환 분류

### DISCUSSION THREAD

**Discussion by SHEYCHAE** [Pleasant-riot/Lung-Disease-Detection#19] [Show and tell]
공기누출. 과다팽창 - 승해
무기폐 . 신생아 호흡곤란증후군 - 광이
폐렴. 흉막삼출 - 훈이

**Comment by SHEYCHAE** [comment]
### 정상 폐 구조
![image](https://user-images.githubusercontent.com/77530039/142984020-6fede72e-acc1-4fad-a9ce-9f5c0692039f.png)

**Reply by JINYUHOON** [reply]
![image](https://user-images.githubusercontent.com/68021998/142988271-de3f729b-d91f-48db-9936-40e79186a2b2.png)

**Reply by SHEYCHAE** [reply]
ㅋㅋ굿!!

**Comment by SHEYCHAE** [comment]
# 공기누출 (기흉, 공기가슴증)
### 정의
- **폐에 생긴 구멍으로 공기가 새면서 늑막강 안에 공기가 차는 질환**
- 새는 공기의 양이 증가할수록 폐는 정상기능 x
- 흉강 안으로 유입되는 공기가 배출되지 않는 경우 양쪽 폐와 심장 사이의 공간과 심장이 한쪽으로 쏠려 응급상황이 발생할 수 있음 

### 진단
- 흉막강 속의 압력이 증가하면서 폐와 심장을 압박함
![image](https://user-images.githubusercontent.com/77530039/142984844-93288abc-4f62-42d6-a3d8-0dfd63be9145.png)
 ( x-ray이미지 : 폐와 심장을 압박하면서 한 쪽으로 쏠리는 현상을 보임)


<img src="https://user-images.githubusercontent.com/77530039/142985477-e6bd09b7-0cec-4353-acf5-b476d31d1af6.png" width="250" height="250"/>
<img src="https://user-images.githubusercontent.com/77530039/142986213-499ee3dd-930d-4855-89af-11dde4e1714c.png" width="250" height="250"/>
 ( x-ray이미지 : 폐 공기누출로 인해 한쪽 폐가 축소된 현상을 보임 )



----------------------------------------------------------------------
### img/ normal_폐 / clahe_img
예시1) 
![image](https://user-images.githubusercontent.com/77530039/142987246-eeacf8e1-5b1d-4ec0-987a-fd6ae78953e2.png)
예시2)
![image](https://user-images.githubusercontent.com/77530039/142988408-71b86ace-dede-4afa-8eff-c571b6544b5d.png)
![image](https://user-images.githubusercontent.com/77530039/142988492-4296805f-1a05-4eb8-970e-390bb47962f7.png)

**Comment by kikiru328** [comment]
# 무기폐
![image](https://user-images.githubusercontent.com/60537388/142987853-6e39c46e-2c11-4fd9-aa23-fa45b9bd71f2.png)

* 폐 또는 폐의 일부가 팽창된 상태를 유지하지 못하고 압축된 상태 *

- 폐의 일부가 팽창된 상태를 유지하지 못하고, 부피가 줄어 압축된 (허탈)을 무기폐라고 함.
- 기관지가 막혀 하부의 폐 조직이 허탈되거나 호흡기 분비물에 의해 기관지가 폐쇄되어 하부의 폐조직이 허탈되는 등 다양한 환경에서 발생함.
- 기관지가 폐색되어 맖초부의 폐조직에 기관지로부터의 공기 유입이 정지, 폐포 내에 공기의 양이 적거나 매우 결핍 된 이상상태를 의미함.

**Comment by SHEYCHAE** [comment]
# 과다팽창 (만성폐쇄성폐질환)
### 정의
- **폐의 과다팽창으로 인해 숨이 차는 증상**
- 폐첨부에 기포질환(폐기종)이 관찰됨
- 폐의 비정상적인 염증반응과 함께 점차 기류제한이 진행됨

- 만성염증은 소기도의 구조를 변화시키는 개형 및 폐쇄를 일으킴
- 염증에 의해 유발되는 폐 실질의 파괴로 폐포가 소기도에 붙어있지 못하게 됨
- 폐 탄성의 감소가 동반되어 결과적으로 호기시(폐에서 공기가 나갈시) 소기도의 내경이 열린 상태를 유지하지 못해 기류제한이 발생

### 진단
![image](https://user-images.githubusercontent.com/77530039/142992367-7f8b72f8-f649-4b5c-a0f9-8d29c38319d5.png)


----------------------------------------------------------------------
### img/ normal_폐 / clahe_img
![image](https://user-images.githubusercontent.com/77530039/142994735-f0f853e6-e4fd-41eb-814c-2fe0d3efb2bb.png)

**Comment by JINYUHOON** [comment]
# 흉막삼출

#### 정의
- 흉막 삼출은 폐의 흉막에서 체액 성분이 스며나오는 삼출 증상이 나타나는 질환. 
- 정상인의 흉막은 폐를 둘러싸고 있는 허파 쪽 흉막과 흉벽 안쪽을 둘러싸고 있는 벽 쪽 흉막으로 이루어져 있음, 두 가슴막 사이의 공간이 흉막 공간
- 정상인의 경우 벽 쪽 흉막에 있는 모세 혈관에서 흉막 안쪽으로 소량의 체액이 일정하게 생성 
#### - 체액의 생성이 많아지는 질환이나 체액의 흡수를 감소시키는 질환이 발생하면 흉막 삼출증이 발생
#### -  흉막 삼출은 원인 질환에 의하여 흉막에서 흡수되는 체액보다 생성되는 체액이 많아질 때 발생

![흉막삼출](https://user-images.githubusercontent.com/68021998/143153523-456d2683-a65b-4965-a3f4-78f63325f5a0.jpg)

- 늑골횡격막강에 균질한 음영 증가 (homogeneous opacity at costophrenic angle)
- 늑골횡격막각의 소실 (loss of costophrenic angle)
- 초승달 징후 (meniscus sign)
- 대엽간열 내부에 고인 흉막삼출액 소견보임
- 흉막비후 : 폐첨부~폐기저부를 따르는 미만성의 얇은 띠

**Comment by JINYUHOON** [comment]
# 폐렴

### 괴사성 폐렴
- 폐렴의 합병증으로 폐 실질 세포들이 죽어가는 (괴사, necrotizing) 과정
- 
치료과정)
![image](https://user-images.githubusercontent.com/68021998/143156271-971ef0a1-3045-4399-955b-2851a4e54969.png)


![image](https://user-images.githubusercontent.com/68021998/143156178-66822d14-128c-46c4-8455-2bc8cebc15c2.png)



# 소아 급성폐렴

### 바이러스성폐렴
- 유아 및 소아에서 하부 호흡기의 바이러스성 폐렴은 폐포 자체보다 폐기질을 주로 먼저 침범하기 때문에, 주된 영상의학적 소견은 기관지벽 비후, 폐문주위/기관지주위 침윤이다
-  흉막삼출, 기흉, 공기낭종, 폐농양은 드물다. 주로 양측성의 반점형 경화 소견을 보이며, 하엽을 많이 침범하고, 공기가둠(air trapping)을 보일 수 있다

![image](https://user-images.githubusercontent.com/68021998/143157417-4130dc3e-9453-4838-b1a0-fc6d06bac341.png)



### 세균성 폐렴

- 가장 흔한 세균성 폐렴의 원인균은 폐렴연쇄구균(Streptococcus pneumoniae)이다.
- 세균성 폐렴은 보통 삼출물과 염증세포가 폐포를 채우면서 경화 소견을 보이는 것이 특징이다. 
- 기관지폐렴은 여러 분절에서 기관지주위 침윤이 주변 폐실질로 퍼지면서 나타나는 반점형 결절이나 경화 소견을 보인다. 
- 흉막삼출이 동반될 수 있다 (상기 흉막삼출 예시 참고)
- 소아에서 특징적으로 둥근 모양의 폐렴(round pneumonia)이 생길 수 있는데, 이는 보통 8세 미만의 어린이에서 드물게 발생하며, 주원인은 폐렴연쇄구균이다. 
- 이는 어린 소아에서 Kohn 모공의 발달이 부족하고 canals of Lambert가 없기 때문에 나타난다. 국소적으로 결절형의 경화를 보이며, 대부분 단발성 병변으로 경계가 좋고 종괴 효과를 가진다
-  폐하엽의 후위(> 80%)와 하위(> 70%)에서 자주 발생

![image](https://user-images.githubusercontent.com/68021998/143157631-6ffc7757-c9a5-4ee0-b07f-ba8171257536.png)



### 마이코플라즈마 폐렴

- 임상 소견이 비전형적이거나 영상 소견이 임상 소견과 맞지 않는 경우 비전형적 폐렴으로 분류하는데, 여기에는 레지오넬라(Legionella), 마이코플라즈마, 클라미디아 등이 흔한 원인균이 된다.
- 영상 소견은 엽성 또는 분절성 경화(37%) 소견이 가장 많았으며, 다음으로 폐문주위/기관지주위 침윤(27%), 국소성 그물결절형 침윤(21%), 반점형 침윤(15%) 순으로 나타났다

![image](https://user-images.githubusercontent.com/68021998/143157777-50631d3e-bf67-4745-a4ce-68808a5a62a7.png)



#### 면역 저하 환자들의 폐렴

예시이미지)
![image](https://user-images.githubusercontent.com/68021998/143157859-f5749348-6e5f-4228-9c44-566a39683c8d.png)


//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

- 진균감염의 특징적인 CT 소견은 폐결절이다. 이런 폐결절은 종종 주변부에 모여있으며, 불분명한 경계를 보이고, 공동형성(cavitation)을 보일 수 있다

![image](https://user-images.githubusercontent.com/68021998/143158019-1b25bea9-a367-42e3-a106-1c9e742fe805.png)


- 폐포자충 폐렴(Pneumocystis pneumonia)은 T 림프구 수치가 낮은 경우나 특정 예방기능이 없는 경우에 주요한 질병과 사망의 원인이 될 수 있다. 흉부 방사선촬영에서 보통 양측성으로 폐문주위에 점진적으로 결합하는 간질 침윤 소견을 보인다. 
- CT에서는 미만성 또는 반점형의 간유리 음영과 소엽간 중격비후를 보인다

![image](https://user-images.githubusercontent.com/68021998/143158205-b40e64b2-f319-4895-abad-62f4853c71de.png)


- 소아 지역사회 폐렴의 합병증으로는 흉막삼출이나 농흉, 기흉, 폐농양, 기관지흉막루(bronchopleural fistula), 괴사성 폐렴 등이 있으며, 이런 합병증은 폐렴으로 입원한 소아 환자의 53%까지 보고된다
- CT에서 폐농양은 폐 실질의 경화성 병변 내에 낭성 영역과 함께 테두리의 조영증강을 보이며, 괴사성 폐렴은 조영증강을 보이지 않는 불균질한 낭성 병변이 경화 내에 위치하지만 테두리 조영증강은 보이지 않는다

![image](https://user-images.githubusercontent.com/68021998/143158350-a118cdc2-13e0-4df2-b830-138defb8e006.png)

**Comment by kikiru328** [comment]
# 신생아 호흡 곤란 증후군 (Neonatal respiratory distress syndrome)
- 신생아 호흡 곤란 증후군은 출생 직후 폐가 팽창하지 못하고 찌그러져 호흡이 제대로 이루어 지지 않는 질환임.
- 폐를 팽창하게 하는 물질인 표면활성제의 생산과 분비가 제대로 이루어 지지 않은 미숙아에게 질환 발현 가능성이 높다.
- 다시 말하면, 폐가 아직 발달하지 않은 상태로 태어난 미숙아는 폐포의 크기가 작음.  흡기시 폐포에 공기 유입이 되면서 팽창해야되는데, 표면활성제의 부재로 폐포가 팽창되지 않을때 증상이 발현된다
![다운로드](https://user-images.githubusercontent.com/60537388/143173981-a2b5da4a-9b70-443e-86b1-65d9c3620c43.png)
.
미숙아의 폐를 보면 우측(RDS)은 좌측(Normal)에 비해 폐가 폐색된 것 처럼 하얗게 질려있음

**Comment by SHEYCHAE** [comment]
아유 다들 잘하셨네요 
칭찬도장 꽝꽝꽝
